### PR TITLE
kops - extensions support

### PR DESCRIPTION
**Pull request by alebabai**
## what
updated manifest to support extensions

## why
#240

### CODE REVIEW

**Comment by osterman** [comment]
Did you test this with `build-manifest`?

**Comment by alebabai** [comment]
```bash
build-kops-manifest $KOPS_TEMPLATE $KOPS_MANIFEST -d extensions=https://raw.githubusercontent.com/cloudposse/geodesic/master/README.yaml
```
[manifest.yaml.txt](https://github.com/cloudposse/geodesic/files/2328494/manifest.yaml.txt)

File: rootfs/usr/local/bin/build-kops-manifest [hunk 1]
```diff
@@ -4,5 +4,5 @@ KOPS_MANIFEST="${2:-$KOPS_MANIFEST}"
 
 mkdir -p `dirname ${KOPS_MANIFEST}`
 
-gomplate -f ${KOPS_TEMPLATE} > ${KOPS_MANIFEST}
+gomplate ${@:3} -f ${KOPS_TEMPLATE} > ${KOPS_MANIFEST}
```
Comment (osterman) [inline rootfs/usr/local/bin/build-kops-manifest]: What does `@:3` do?

Comment (osterman) [inline rootfs/usr/local/bin/build-kops-manifest]: Let's name this parameter the way we do the others.

File: rootfs/templates/kops/default.yaml [hunk 2]
```diff
@@ -138,3 +138,9 @@ spec:
   {{- range (getenv "KOPS_AVAILABILITY_ZONES" | strings.Split ",") }}
   - {{.}}
   {{- end }}
+    
+---
```
[the remark this answers is not in the document]
Comment (alebabai) [reply rootfs/templates/kops/default.yaml]: done

**Pull request merged**